index.js

```diff
@@ -133,9 +133,9 @@ async function send (ctx, path, opts = {}) {
     }
     ctx.set('Cache-Control', directives.join(','))
   }
-  ctx.type = type(path)
   ctx.body = fs.createReadStream(path)
-
+  ctx.type = ctx.res.type ? ctx.res.type : type(path);
```

Comment:
I think if anyone is setting the content type manually, then it should be set to this. otherwise it should be evaluated on the basis of file extension. 
Can you please suggest with others in the Koa panel for this. They might come up with the proper explanation.